src/test/java/com/microsoft/jenkins/appservice/integration/IntegrationTest.java

```diff
@@ -89,8 +89,8 @@ To execute the integration tests you need to set the credentials env variables (
             appServicePlanName = TestEnvironment.loadFromEnv("APP_SERVICE_TEST_DEFAULT_APP_SERVICE_PLAN_NAME_PREFIX", "appserviceplan") + "-" + TestEnvironment.GenerateRandomString(16);
 
             SkuDescription sd = new SkuDescription();
-            sd.withTier(TestEnvironment.loadFromEnv("APP_SERVICE_TEST_DEFAULT_PRICING_TIER_SKU", "FREE"));
-            sd.withSize(TestEnvironment.loadFromEnv("APP_SERVICE_TEST_DEFAULT_PRICING_TIER_SIZE", "F1"));
+            sd.withTier(TestEnvironment.loadFromEnv("APP_SERVICE_TEST_DEFAULT_PRICING_TIER_SKU", "SHARED"));
```

Comment:
why this change?

Reply:
free tier only support 10 service plan, I added more test cases which causes total plan counts larger than 10

Reply:
Shared tier may charge a little fee, but it is ok since we will destroy the resources as soon as the tests finish